presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveCreateTable.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import io.prestosql.tempto.ProductTest;
+import org.testng.annotations.Test;
+
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static io.prestosql.tests.utils.QueryExecutors.onHive;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+
+public class TestHiveCreateTable
+        extends ProductTest
+{
+    @Test(groups = STORAGE_FORMATS)
+    public void testCreateTable()
+    {
+        onPresto().executeQuery("CREATE TABLE test_create_table(a bigint, b varchar, c smallint) WITH (format='ORC')");
+        onPresto().executeQuery("INSERT INTO test_create_table(a, b, c) VALUES " +
+                "(NULL, NULL, NULL), " +
+                "(-42, 'abc', SMALLINT '-127'), " +
+                "(9223372036854775807, 'abcdefghijklmnopqrstuvwxyz', SMALLINT '32767')");
+        assertThat(onPresto().executeQuery("SELECT * FROM test_create_table"))
+                .containsOnly(
+                        row(null, null, null),
+                        row(-42, "abc", -127),
+                        row(9223372036854775807L, "abcdefghijklmnopqrstuvwxyz", 32767));
+        assertThat(onHive().executeQuery("SHOW TBLPROPERTIES test_create_table"))
+                .contains(row("transactional", "false"));
+        onPresto().executeQuery("DROP TABLE test_create_table");
+    }
+
+    @Test(groups = STORAGE_FORMATS)
```

Comment:
PROFILE_SPECIFIC? Hive 2 is not going to pass `.contains(row("transactional", "false"));` Or maybe create separate test for that table property.

Reply:
It is, because we set this flag unconditionally, regardless of Hive version.

Reply:
Ah, the problem is:

- if i don't set `transactional`, HMS adds `transactional=true`
- if i set `transactional=false`, HMS ... removes it (and doesn't add `transactional=true`)

so in the test I need to expect the property is not set or is set to `false` :/ :/ :/ :/ :/ :/ :/